src/main/deploy/swerve/modules/backleft.json

```diff
@@ -0,0 +1,27 @@
+{
+  "location": {
+    "front": 0,
+    "left": 0
+  },
+  "absoluteEncoderOffset": 0,
+  "drive": {
+    "type": "sparkflex",
+    "id": 0,
```

Comment:
You can start counting up the CAN IDs from 0, starting anew for each different device type, e.g. 0-3 for the four drive talonfx, 0-3 for the four angle talonsrx, and 0 for the pigeon2.